app/assets/v2/js/pages/bounty_details.js

```diff
@@ -427,6 +436,9 @@ window.addEventListener('load', function() {
                     var is_interested = is_on_interest_list(result['pk']);
                     update_interest_list(result['pk']);
 
+                    // // Find fulfiller information
+                    // update_fulfiller_list(result['pk']);
```

Comment:
this is intended to remain commented out, no?

Reply:
Yup -- fixed.